src/Microsoft.DotNet.CodeFormatter.Analyzers.Tests/ExplicitVariableTypeAnalyzerTests.cs

```diff
@@ -0,0 +1,399 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.CodeFormatter.Analyzers;
+
+using Xunit;
+
+namespace Microsoft.DotNet.CodeFormatter.Analyzers.Tests
+{
+    public sealed class ExplicitVariableTypeAnalyzerTests : AnalyzerFixerTestBase
+    {
+        public ExplicitVariableTypeAnalyzerTests()
+        {
+            DisableAllDiagnostics();
```

Comment:
very likely disabling all diagnostics is something common to these tests that could be in a test base class. something to think about for a future change